--- libgsf-1.14.25/python/Makefile.am~	2011-12-08 00:05:18.000000000 +0100
+++ libgsf-1.14.25/python/Makefile.am	2013-02-11 23:39:32.562031881 +0100
@@ -15,7 +15,7 @@ _gsfmodule_la_CFLAGS = 		\
 	-I$(top_srcdir)		\
 	$(LIBGSF_CFLAGS)
 _gsfmodule_la_LDFLAGS = -module -avoid-version -export-symbols-regex init_gsf
-_gsfmodule_la_LIBADD = $(top_builddir)/gsf/libgsf-1.la $(LIBGSF_LIBS)
+_gsfmodule_la_LIBADD = $(top_builddir)/gsf/libgsf-1.la $(LIBGSF_LIBS) -lpython2.7
 _gsfmodule_la_SOURCES = gsfmodule.c
 nodist__gsfmodule_la_SOURCES = gsf.c
 CLEANFILES += gsf.c
